### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by mjs**
Merge latest upstream in jump fork. This was quite a difficult merge with many conflicts so some careful review would be appreciated.

### CODE REVIEW

File: FluentTerminal.sln [hunk 1]
```diff
@@ -19,7 +19,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FluentTerminal.App.Services
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FluentTerminal.App.Services.Test", "FluentTerminal.App.Services.Test\FluentTerminal.App.Services.Test.csproj", "{B353EAF4-8332-4C89-B986-B0B29F7E21FC}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FluentTerminal.App.ViewModels", "FluentTerminal.App.ViewModels\FluentTerminal.App.ViewModels.csproj", "{44E8E225-F46C-47EC-9290-FFA08FCCF1B5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FluentTerminal.App.ViewModels", "FluentTerminal.App.ViewModels\FluentTerminal.App.ViewModels.csproj", "{44E8E225-F46C-47EC-9290-FFA08FCCF1B5}"
```
Comment (felixse) [inline FluentTerminal.sln]: this doesn't look good

File: FluentTerminal.App/Dialogs/SshInfoDialog.xaml [hunk 2]
```diff
@@ -92,13 +97,14 @@
...
-                        <DataTemplate>
-                            <TextBlock Text="{Binding Converter={StaticResource LineEndingStyleConverter}}" 
+                        <DataTemplate x:DataType="enums:LineEndingStyle">
+                            <TextBlock Text="{x:Bind Converter={StaticResource I18NConverter}, ConverterParameter=LineEndingStyle}" 
                                        HorizontalAlignment="Stretch"/>
                         </DataTemplate>
                     </ComboBox.ItemTemplate>
                 </ComboBox>
                 <TextBlock Grid.Column="0" Grid.Row="1"
+                           x:Uid="MoshPorts"
```
Comment (felixse) [inline FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: I added a missing colspan somewhere below here, fixed some layout issue but looks like it got lost

File: FluentTerminal.sln (hunk 1 above)
Comment (mjs) [reply FluentTerminal.sln]: I'll take a look.

File: FluentTerminal.App/Dialogs/SshInfoDialog.xaml (hunk 2 above)
Comment (mjs) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: I'll figure this out too

File: FluentTerminal.sln (hunk 1 above)
Comment (mjs) [reply FluentTerminal.sln]: Fixed

File: FluentTerminal.App/Dialogs/SshInfoDialog.xaml (hunk 2 above)
Comment (mjs) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: Sorry, I've compared SshInfoDialog.xaml between the felixse master and the jump master and I don't see the missing colspan. Are you able to provide some more detail?

Comment (mjs) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: Never mind... I think I've found it. Looks like it was for the line ending style combobox. Fixing.

Comment (felixse) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: compare the ComboBox starting in line 94 with https://github.com/felixse/FluentTerminal/blob/master/FluentTerminal.App/Dialogs/SshInfoDialog.xaml#L89
Also the Converter used here is no longer needed.

Comment (felixse) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: you might want to delete the converter as well

Comment (mjs) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: I've just pushed the fix (definitely makes the dialog more usable). There was also a "x:uid" which had been unintentionally removed (fixed too).

Re the converter, I believe @peske added that after the last merge of SSH functionality to the main repo. It appears to be necessary now. See https://github.com/jumptrading/FluentTerminal/commit/d24fae6453eebda3fb502eb200330b0438955c74

Comment (felixse) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: The converter was already part of the merge to upstream some days ago. But I refactored it to use the new I18NConverter to simplify it and offer localization

Comment (mjs) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: Got it. I'll sort that out now.

Comment (mjs) [reply FluentTerminal.App/Dialogs/SshInfoDialog.xaml]: All fixed. Merging now.

**Pull request merged**